pkgs/servers/mail/smtp-to-telegram/default.nix

```diff
@@ -0,0 +1,24 @@
+{ buildGoPackage, fetchFromGitHub, lib }:
+
+buildGoPackage rec {
+  name = "smtp-to-telegram";
+  rev = "8092fc365b902d160d3e3de6ceb7ca67a17392d0";
```

Comment:
Can we fetch a release instead?

Reply:
Unfortunately, there hasn't been any release recently and the older releases are somewhat behind.

Reply:
Then we need to change the version to the commit date.

Reply:
done 🙂